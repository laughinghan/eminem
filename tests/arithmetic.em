let .x 1
let .y 2
print "x = " x "; y = " y "
"
print "x + y = " (x.plus y) "
"
let .add3 (3.plus)
print "(add3 x) = " (add3 x) "
"
