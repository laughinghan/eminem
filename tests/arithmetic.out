x = 1; y = 2
x + y = 3
(add3 x) = 4
